build/setup.d/10-upgrade-base.sh

```diff
@@ -54,7 +54,7 @@ popd
 
 # install 3.16. LTS kernel and make sure it updates to the last version
 # also this step should build ixgbevf and ena kernel modules and put them into initramfs
-apt-get install -y linux-image-virtual-lts-xenial
+apt-get install -y linux-image-generic-lts-xenial
```

Comment:
virtual is the minimalistic kernel. This lead to issues compiling the nvidia kernel drivers